src/vscode/src/.gitignore

```diff
@@ -0,0 +1,8 @@
+out
+dist
+node_modules
+.vscode-test/
+*.vsix
+
+# Parent .gitignore excludes 'debug' directories, but this one is legit
+!src/debug
```

Comment:
💡 just a not that we can remove this line, but we can do this latter.

Reply:
yes, we can remove once we move vscode to separate repo :)